ledger/participant-integration-api/src/main/scala/platform/store/appendonlydao/JdbcLedgerDao.scala

```diff
@@ -1143,7 +1143,7 @@ private[platform] object JdbcLedgerDao {
         serverRole,
         jdbcUrl,
         connectionPoolSize,
-        250.millis,
```

Comment:
250 millis is the minimum setting, hikari does not allow lower, if we need to increase this in some deployment scenarios that means that the query load + max connection pool size + this timeout combination is insufficient, (and I guess: increasing the connection pool size is not desirable).
So how about making this also a configuration parameter instead, so we can play around with this parameters per deployment?

Reply:
alright in the next commit extracted this value out to something configurable -- it does lead to a lot of repetition of the configuration in multiple places (we may be able to do further defaulting). https://github.com/digital-asset/daml/pull/9750/commits/c193e875b62cfaffca5242a20aec28242b07e4e8

Reply:
wow, that exploded! sorry for that and thanks for doing it :raised_hands: